src/main/kotlin/lotto/model/WinningNumber.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.model
+
+class WinningNumber {
+    fun makeMatchNumber(numberGenerator: NumberGenerator): LottoTicket {
+        val matchNumber = LottoTicket().makeLottoTicket(numberGenerator)
+        return matchNumber
+    }
+
+    fun makeBonusNumber(numberGenerator: NumberGenerator): Int {
+        val bonusNumber = numberGenerator.generateRandomNumber()
+        return bonusNumber
+    }
```

Comment:
코드 스타일의 영역이겠지만, 해당 코드는 다음과 같이 간추려 질 수 있습니다.

```suggestion
    fun makeBonusNumber(numberGenerator: NumberGenerator): Int = 
        numberGenerator.generateRandomNumber()
```